javascript-algorithms-and-data-structures/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/index.js

```diff
@@ -24,108 +18,113 @@ const categories = {
   560: { category: "Backend Development", className: "backend" },
 };
 
-const forumCategories = (id) => {
-  return categories.hasOwnProperty(id)
-    ? `<a class="category ${categories[id]["className"]}" target="_blank" href="${FORUM_CATEGORY_URL}${categories[id]["className"]}/${id}">${categories[id]["category"]}</a>`
-    : `<a class="category general" target="_blank" href="${FORUM_CATEGORY_URL}general/1">General</a>`;
-};
+async function fetchData() {
+  const res = await fetch(FORUM_LATEST);
+  const data = await res.json();
+  showLatestPosts(data);
+}
+fetchData();
```

Comment:
`fetchData()` works here, though it might be better to move this function call to the end of this file. Then it should be a little more clear that calling the `fetchData` function kicks off the rest of the code.

Or if you'd prefer to leave `fetchData()` here, a newline between the function call and the function itself will help with readability.